bids/events/base.py

```diff
@@ -250,24 +250,25 @@ def __init__(self, base_dir, default_duration=0, default_amplitude=1,
         self.extra_columns = extra_columns
         self.sampling_rate = sampling_rate
 
-    def read(self, files=None, reset=True, **kwargs):
+    def read(self, file_directory=None, reset=True, **kwargs):
         ''' Read in and process event files.
         Args:
-            files (None, list): Optional list of event files to read from. If
-                None (default), will use all event files found within the
-                current BIDS project. Note that if `files` is passed, existing
-                event files within the BIDS project are completely ignored.
+            file_directory (None, str): Optional path to folder containing
```

Comment:
I think I would prefer to keep some sort of files argument. We could also fetch the files and put them all in a temp folder, and use the same approach I'm using. Not the most efficient but I think it's a useful feature. I rather not overwrite the original dataset if I can help it.